mmaction/datasets/pipelines/augmentations.py

```diff
@@ -1120,6 +1127,11 @@ def __call__(self, results):
         results['flip'] = flip
         results['flip_direction'] = self.direction
 
+        if self.label_transforms is not None and flip:
+            label = results['label']
```

Comment:
```python
results['label'] = self.label_transforms.get(label, results['label'])
```

Reply:
okay my fault, i actually meant `results['label'] = self.label_transforms.get(results['label'], results['label']))` to save one line. sigh...

Reply:
I thought it would make the code easier to read.....